src/integrationTest/java/uk/gov/hmcts/darts/event/service/impl/InterpreterUsedHandlerTest.java

```diff
@@ -60,6 +64,8 @@ void throwsOnUnknownCourthouse() {
 
     @Test
     void handlesScenarioWhereCourtCaseAndHearingDontExist() {
+        CourtroomEntity courtroom = dartsDatabase.createCourtroomUnlessExists(SOME_COURTHOUSE, SOME_ROOM);
+        nodeRegisterStub.setupNodeRegistry(courtroom);
```

Comment:
if this is common for all the tests, shall we move it in the test setup logic (`setupStubs`)?

Reply:
done